src/main/java/net/gnomeffinway/depenizen/support/bungee/BungeeTag.java

```diff
@@ -0,0 +1,36 @@
+package net.gnomeffinway.depenizen.support.bungee;
+
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.tags.Attribute;
+import net.aufdemrand.denizencore.tags.ReplaceableTagEvent;
+import net.aufdemrand.denizencore.tags.TagManager;
+import net.aufdemrand.denizencore.utilities.CoreUtilities;
+import net.gnomeffinway.depenizen.Settings;
+import org.bukkit.event.Listener;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public class BungeeTag implements Listener {
+
+    public BungeeTag(JavaPlugin depenizen) {
+        depenizen.getServer().getPluginManager().registerEvents(this, depenizen);
```

Comment:
I was just following the example from ServerTags (which mcmonkey told me to use).  What should it implement, if anything?